app/controllers/users_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  before_action :set_user
+
+  rescue_from ActionPolicy::Unauthorized do |ex|
+    redirect_to root_path, alert: ex.result.message
+  end
+
+  def edit
```

Comment:
https://actionpolicy.evilmartians.io/#/./rails?id=verify_authorized-hooks 
allows to DRY authorize!